src/main/java/org/opentripplanner/index/model/TripShort.java

```diff
@@ -15,6 +15,9 @@
     public AgencyAndId serviceId;
     public String shapeId;
     public Integer direction;
+    public String drtMaxTravelTime;
+    public String drtAvgTravelTime;
+    public Double drtAdvanceBookMin;
 
```

Comment:
null fields will not be serialized, or need not be serialized. However it does seem strange to include these in every object when they will be very rarely used. This might be a good case for nesting a DRTTripDetails. Then again, I'm not sure this is relevant information for a "short" representation of a trip. It depends on how this Index API response is intended to be used. Once a trip is known to exist from the short form, you could look up the full details of that individual trip in a separate API call.

Reply:
Fair points. I'll remove those fields from the TripShort representation.